### ISSUE TITLE
Hardcoded dependency to lux

### ISSUE THREAD

**Issue by georgringer** [in2code-de/luxletter#93]
The `NewsletterController` got a hardcoded dependency to lux with

```php
$visitorRepository = ObjectUtility::getObjectManager()->get(VisitorRepository::class);
```

**Comment by einpraegsam** [comment]
Yes I know. Any problems with this? Or: How can I reproduce an error with this? Or is it just a code cleanup proposal?

**Comment by georgringer** [comment]
I don't know yet as I currently don't really need the backend modules but I will try to check

**Comment by georgringer** [comment]
I don't know when the ajax route for `receiverDetailAjax` is called but maybe only if lux is installed? however still a bad code small but fully understandable if you close the issue ;)

**Comment by pixeldesu** [comment]
We looked at the code in question, and the specific AJAX endpoint that is defined here is only ever called from Lux, but only through JavaScript code that is loaded from Luxletter, meaning that without Luxletter, there is no call to it even (little to no potential for an exception).

That entire getup is also making it harder to refactor it out _simply_ because we might just end up switching the hardcoded dependency the other way around (if we require Luxletter inside Lux).

This requires some more consideration and for the meantime is probably not feasible, so we'll close this issue.

Thank you for the report though and caring about code quality! 🎉
